### ISSUE TITLE
Interactive Learning ouput stories file shows unicode code not character string

### ISSUE THREAD

**Issue by HatuneMiku** [RasaHQ/rasa#3564]
**Rasa version**:
`Rasa 1.0.1`
**Python version**: 
`Python 3.6.8`
**Operating system** (windows, osx, ...):
`Windows 10 Pro 1903`
**Issue**:

Interactive Learning output stories file shows unicode code but not character string, 
like I have submitted in #2499.

This is a story by Interactive Learning ouput:

```
## Generated Story -4851626544216716596
* morning_report
    - action_get_user_info
    - slot{"family_members": [{"\u8001\u5a46": 614}]}
    - slot{"account_name": "\u63a5\u73ed\u4eba"}
    - action_identification_ask
* exit
    - utter_exit_record
    - action_slot_reset
    - reset_slots
```

I have try this code:
```
a = "\u5317\u4eac"
print(a)

with open("test.md", "w", encoding="utf8") as file:
    file.write("\u5317\u4eac")

```
and I get "北京" with the print function and in "test.md" file. I don't konw why this happend. 
thanks a lot.

**Comment by HatuneMiku** [comment]
And maybe it will influence the visualization of the stories.  All these unicode code like `"\u5317\u4eac"` will shows `��������` in visualization output file graph.html, but I am not very sure, becasue in `./rasa/core/training/visualization.html` some code not use "utf8", so the file encode will depended by system. like line 288:

```
def persist_graph(graph, output_file):
    """Plots the graph and persists it into a html file."""
    import networkx as nx

    expg = nx.nx_pydot.to_pydot(graph)

    with open(visualization_html_path(), "r") as file:
        template = file.read()

    # customize content of template by replacing tags
    template = template.replace("// { is-client }", "isClient = true", 1)
    template = template.replace(
        "// { graph-content }", "graph = `{}`".format(expg.to_string()), 1
    )

    with open(output_file, "w") as file:
        file.write(template)
```

**Comment by akelad** [comment]
Thanks for raising this issue, @erohmensing  will get back to you about it soon.

**Comment by erohmensing** [comment]
Hi @HatuneMiku. I'm sorry this doesn't seem to have been resolved after a long time asking for it. I've checked into it, here's somehting interesteing: 
```
## Generated Story 1132402095665157564
* 北京{"\u5317\u4eac": "\u5317\u4eac"}
    - 北京
```
This only seems to happen if the characters are inside entities or slots. My instict is that this is happening because of some sort of `json.loads` or `json.dumps` that isn't being encoded, since the regular output is being encoded. I'll look into a fix for you.

**Comment by HatuneMiku** [comment]
Thanks @erohmensing , If we are using json to save characters are inside entities or slots, maybe this can help:

```
import json
strings = "北京"
print(json.dumps(strings))
print(json.dumps(strings, ensure_ascii=False))
```
the result are:
```
"\u5317\u4eac"
"北京"
```

**Comment by erohmensing** [comment]
Thanks, I actually found that already, I've fixed it locally. Glad we came to the same conclusion :) I'm working right now on the visualization. On my side, characters show up fine, are you still having issues with visualization?

**Comment by HatuneMiku** [comment]
It is better to use utf-8 everywhere just like the fix for #2539 in rasa_core, so maybe we should add `encoding=utf-8` to the open function or IO function in visualization.
